pallets/currency/Cargo.toml

```diff
@@ -22,9 +22,9 @@ pallet-transaction-payment = { git = "https://github.com/paritytech/substrate",
 primitives = { package = "spacewalk-primitives", path = "../../primitives", default-features = false }
 
 # Orml dependencies
-orml-tokens = { git = "https://github.com/open-web3-stack/open-runtime-module-library", branch = "polkadot-v0.9.37", default-features = false }
-orml-currencies = { git = "https://github.com/open-web3-stack/open-runtime-module-library", branch = "polkadot-v0.9.37", default-features = false }
-orml-traits = { git = "https://github.com/open-web3-stack/open-runtime-module-library", branch = "polkadot-v0.9.37", default-features = false }
+orml-tokens = { version = "0.4.1-dev", default-features = false }
+orml-currencies = { version = "0.4.1-dev", default-features = false }
+orml-traits = { version = "0.4.1-dev", default-features = false }
```

Comment:
Is it really necessary to change all these dependencies to the higher version just to patch them down again? Why don't we just leave the declarations we have already? Is there an error if we leave the version declaration as it was?

Reply:
@ebma 
yes. It is an error if we leave them as before.
There error looks like that "you do not use patch 0.4.1-dev in your packages" and it failed to compile.

Reply:
Aha okay hmm. Can you try the trick with the double slash? You might have noticed that in the existing `patch` statements many URL declarations have an 'unnecessary' double slash. Maybe if you use that, ie. 
```
- orml-tokens = { git = "https://github.com/open-web3-stack/open-runtime-module-library", branch = "polkadot-v0.9.37", default-features = false }
- orml-currencies = { git = "https://github.com/open-web3-stack/open-runtime-module-library", branch = "polkadot-v0.9.37", default-features = false }
- orml-traits = { git = "https://github.com/open-web3-stack/open-runtime-module-library", branch = "polkadot-v0.9.37", default-features = false }

+ orml-traits = { git = "https://github.com/open-web3-stack//open-runtime-module-library", branch = "polkadot-v0.9.37" }
+ orml-currencies = { git = "https://github.com/open-web3-stack//open-runtime-module-library", branch = "polkadot-v0.9.37" }
+ orml-tokens = { git = "https://github.com/open-web3-stack//open-runtime-module-library", branch = "polkadot-v0.9.37" }
```
the compilation error disappears and we can get rid of all the extra changes?

Reply:
Yes. Thanks. It works. Even more. Now it works with one slash to.))

Reply:
@ebma 
Done.

Reply:
@ebma but anyway
you wrote that we also need use dependency from crates for orml packages. so i added dependency to crates instead of git.